consoleme/handlers/v2/audit.py

```diff
@@ -13,6 +13,13 @@
 credential_mapping = CredentialAuthorizationMapping()
 
 
+def _get_last_page(total, page_size):
+    pages = total / page_size
```

Comment:
Does this need to be `int(total / page_size)` ? thinking of a situation like `_get_last_page(10, 999)`

Reply:
Ah yeah good catch, I forgot that Python doesn't do integer division when you divide two ints